src/frontend/utils/initWebinarContext.ts

```diff
@@ -15,7 +15,12 @@ export const initWebinarContext = async (video: Video) => {
   if (useLiveRegistration.getState().liveRegistration) return;
 
   let anonymousId: Maybe<string>;
-  if (!checkLtiToken(getDecodedJwt())) {
+
+  const decodedJwt = getDecodedJwt();
+  if (decodedJwt.user?.anonymous_id !== undefined) {
+    anonymousId = decodedJwt.user?.anonymous_id!;
+    setAnonymousId(anonymousId);
+  } else if (!checkLtiToken(decodedJwt)) {
```

Comment:
```suggestion
  const decodedJwt = getDecodedJwt();
  let anonymousId = decodedJwt.user?.anonymous_id;
  if (anonymousId) {
    setAnonymousId(anonymousId);
  } else if (!checkLtiToken(decodedJwt)) {
```

This change can be made only if the `anonymous_id` type in the token is no more `Nullable`. So if you ensure it is not Nullable, then you can make this change

Reply:
yes nullable had nothing to come here